specification/nodes/_types/Stats.ts

```diff
@@ -17,7 +17,7 @@
  * under the License.
  */
 
-import { IndexStats } from '@indices/stats/IndexStats'
+import { IndexStats } from '@indices/stats/types'
```

Comment:
same here, especially if used from a different namespace, should go into `indices/_types`

Reply:
It's used only here :)